pom.xml

```diff
@@ -125,6 +125,11 @@
 			<artifactId>mockito-inline</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.mockk</groupId>
+			<artifactId>mockk-jvm</artifactId>
+			<version>1.13.5</version>
```

Comment:
Add scope test for this library

Reply:
Removed as it was used in intermediate experimentation :)